index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: finish mobile versio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <section id="works" class="section-works">
+            <!-----card-works for Tonic------->
+            <div class="card-works tonic">
+                    <img src="./images/SnapshootPortfolio.png" alt="Portfolio Snapshot" class="card-work">
+    
+                <div class="left-block">
+                    <div class="project">
+                        <div class="project-title">                         
+                            <a href="#" class="title">Tonic</a>
+                        </div>
+                        <div class="frame-2">
+                            <a href="#" class="canopy btn-link"> Canopy</a>
+                            <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+                            <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+                            <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                            <a href="#" class="year btn-link"> 2015 </a>
+                        </div>
+                    </div>
+                    <div class="work-detail">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+    
+                    <div class="tags">
+                        <span class="tag">html</span>
+                        <span class="tag">css</span>
+                        <span class="tag">javascript</span>
+                    </div>
```

Comment:
- [ ] kindly please use ul and li for categories in the "Work" section. Html, CSS, and Javascript should be list items as required in the project requirement, check below the format :grinning:  :grinning:  :grinning: 

```
<ul class="flexbox-container">
    <li class="flex-item item1">css</li>
    <li class="flex-item item2">html</li>
    <li class="flex-item item3">bootstrap</li>
    <li class="flex-item item4">Ruby</li>
</ul>
```